src/electron/flux/action/tab-stops-action-creator.ts

```diff
@@ -2,19 +2,50 @@
 // Licensed under the MIT License.
 
 import { TabStopsActions } from 'electron/flux/action/tab-stops-actions';
+import { DeviceFocusController } from 'electron/platform/android/device-focus-controller';
 
 export class TabStopsActionCreator {
-    constructor(private readonly tabStopsActions: TabStopsActions) {}
+    constructor(
+        private readonly tabStopsActions: TabStopsActions,
+        private readonly deviceFocusController: DeviceFocusController,
+    ) {}
 
     public enableTabStops = () => {
```

Comment:
Consider the following scenario:

1. User clicks the toggle to start focus tracking
2. `tabStopsActions.enableFocusTracking` is invoked and the store gets updated
3. `deviceFocusController.enableFocusTracking` fails because the device is disconnected

Once the DeviceDisconnectedPopup goes away, `TabStopsStore.focusTracking` thinks tab stops are enabled on the device but the device never received the command.

What if, instead of having the TabStopsActionCreator call the DeviceFocusController, the DeviceFocusController called the TabStopsActionCreator? That would cause the TabStopsStore to change only when the device was changed, and would allow us to remove the functions that are just one-line wrappers around the DeviceFocusController